config/mbed/CMakeLists.txt

```diff
@@ -239,6 +239,7 @@ target_include_directories(chip INTERFACE
     ${CHIP_ROOT}/src/lib
     ${CHIP_ROOT}/src/lib/core
     ${CHIP_ROOT}/third_party/nlassert/repo/include
+    ${CHIP_ROOT}/third_party/capsense
```

Comment:
I don't think it belong in the CMakeList.txt that build CHIP. It belongs in the CMakeLists.txt of each example. 
This also require update of the shell scripts that link sources in `third_party`.

Reply:
Done